x-pack/plugins/ml/public/explorer/explorer_charts/explorer_chart_distribution.js

```diff
@@ -414,31 +417,69 @@ export class ExplorerChartDistribution extends React.Component {
       if (_.has(marker, 'anomalyScore')) {
         const score = parseInt(marker.anomalyScore);
         const displayScore = (score > 0 ? score : '< 1');
-        contents += `anomaly score: ${displayScore}`;
+        contents += intl.formatMessage({
```

Comment:
FYI @azasypkin because this code is complex we concatenate a lot of separate labels instead of using one long label

Reply:
Yeah, that's a complex one. In some cases (like in my comment below) it makes sense to use `plural`, in some cases it makes sense to use `select`, based on `marker` or something else, e.g. (synthetic)
```js
intl.formatMessage({
	id: 'xpack.ml.explorer.distributionChart.valueLabel',
  	defaultMessage: '{br}value: {value}' + 
  		'{typicalValue, select, n/a {} other {{br}typical: {typicalValue}}}'
}, {
	br: '<br />',
	value: formatValue(marker.value, config.functionDescription, fieldFormat),
	typicalValue: typeof marker.numberOfCauses === 'undefined' || marker.numberOfCauses === 1 
		? formatValue(marker.typical, config.functionDescription, fieldFormat)
		: 'n/a'
});
```
Single label can include a couple of selectors, but please don't make it too complex as translators won't be able to follow that eventually will lead to errors/typos/etc.

Use your and reviewer's judgement, it's hard to tell what is appropriate here and what isn't without a domain knowledge, I'd say I'm much more concerned about the place where this semi-auto-generated HTML string is injected and whether it's properly escaped and safe to insert :skull: